wgengine/magicsock/magicsock.go

```diff
@@ -52,6 +52,12 @@ type Conn struct {
 	logf          func(format string, args ...interface{})
 	sendLogLimit  *rate.Limiter
 
+	// bufferedIPv4From and bufferedIPv4Packet are owned by
+	// ReceiveIPv4, and used when both a DERP and IPv4 packet arrive
+	// at the same time. It stores the IPv4 packet for use in the next call.
+	bufferedIPv4From   *net.UDPAddr // if non-nil, then bufferedIPv4Packet is valid
+	bufferedIPv4Packet []byte       // re-used memory, copy of it.
```

Comment:
I don't understand the end of the comment sentence. do you mean this is a copy of re-used memory, or that the user of it must copy from it?

Reply:
From the rest of the code, I think it means the eventual consumer of this buffered packet must copy out of this buffer rather than pass by reference. Not quite sure how to phrase it, maybe "owned by Conn, readers must copy from it" ?

Reply:
It was a typo; I reworded in the follow-up replacement commit.

By default, "owned by" also means owns the underlying buffer for slices, so I don't think we need to clarify more ("copy out of it").